addons/xbmc.gui/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.gui" version="5.13.0" provider-name="Team Kodi">
-  <backwards-compatibility abi="5.13.0"/>
+<addon id="xbmc.gui" version="5.13.1" provider-name="Team Kodi">
+  <backwards-compatibility abi="5.13.1"/>
```

Comment:
bumping the backwards compatibility in beta stage is not something that is acceptable. This will break every skin in repo

Reply:
ok, no need to bump backwards compatibility. this change just removes dead code

Reply:
if they keep working then keeping it at 5.13.0 should be fine.